app/src/main/res/values-de/strings.xml

```diff
@@ -885,4 +978,18 @@
 
     <!-- This string is displayed in the content language list in front of the system default language in between parenthesis -->
     <string name="language_default">Standard</string>
+
+    <!-- This string is displayed in the URL bar when the user is in the home page.
+         '%1$s' will be replaced at runtime with the app's name. -->
+    <string name="url_home_title">%1$s-Startseite</string>
```

Comment:
Is the "-" specific to some languages? I see that most of them don't use it but some do.